src/components/PageSections/Pagination/Filter.js

```diff
@@ -0,0 +1,389 @@
+import React, { useState, useReducer, useEffect } from 'react';
+import DayPicker, {LocaleUtils} from "react-day-picker";
+import classNames from 'classnames';
+import { useIntl } from 'react-intl';
+import { useDesktopQuery } from 'js/helpers/reactMediaQueries';
+import { filter as i18n1 } from 'js/i18n/definitions';
+import { mobilePopupHelper } from 'js/helpers/hooks';
+import { months, weekdaysLong, weekdaysShort } from 'js/i18n/constants';
+
+const Filter = ({applyFilter, fromDate, toDate, lowerBound, upperBound}) => {
+  const intl = useIntl();
+  const isDesktop = useDesktopQuery();
+
+  if (!isDesktop) {
+    return (
+      <FilterMobilePopup
+        applyFilter={applyFilter}
+        fromDate={fromDate}
+        toDate={toDate}
+        lowerBound={lowerBound}
+        upperBound={upperBound}
+      />
+    )
+  } else {
+    return (
+      <div className="coa-filter__container col-md-auto">
+        <span className="coa-filter__rail_label">{intl.formatMessage(i18n1.filter)}</span>
+        <FilterBox
+          applyFilter={applyFilter}
+          fromDate={fromDate}
+          toDate={toDate}
+          lowerBound={lowerBound}
+          upperBound={upperBound}
+        />
+      </div>
+    )
+  }
+}
+
+const FilterMobilePopup = ({applyFilter, fromDate, toDate, lowerBound, upperBound}) => {
+  const intl = useIntl();
+  const [menuOpened, setMenuOpened] = useState(false);
+  mobilePopupHelper(menuOpened, setMenuOpened)
+
+  return (
+    <div style={{width: "100%"}}>
```

Comment:
what is the default width?

Reply:
It looks like me adding "width:100%" makes no difference! It did at one point in development, but now everything has changed around it so our container always fills the full width of the screen.